src/systems/tracked-controls.js

```diff
@@ -22,20 +24,21 @@ module.exports.System = registerSystem('tracked-controls', {
   },
 
   tick: function () {
-    this.updateControllerList();
+    var gamepads;
+    // Call getGamepads for Chrome.
+    gamepads = navigator.getGamepads && navigator.getGamepads();
```

Comment:
how does this help if it's `navigator.getGamepads()` is still getting called on every tick?

why not [keep this](https://github.com/aframevr/aframe/pull/3112/files#diff-313911aa60d8456ddbf3aa74faa9368cL38) in `updateControllerList` (which is now throttled)?

Reply:
Some backstory, that @ngokevin @dmarcos and I ran into last time around - if you don't call `navigator.getGamepads()`, then if you are using Chromium/Chrome (which includes derivatives like Oculus Browser) you won't get gamepad updates until the next one.  Firefox doesn't care, but on several prominent platforms (GearVR, Daydream) the only native WebVR browser options are Chromium/Chrome-based.  As I mentioned, I don't recall if we characterized Edge in this regard.

Reply:
thanks - yep, I know that about Chrome. that inconsistency in browser implementations is what this spec issue is for: w3c/gamepad#8

what I'm suggesting to do is move the `navigator.getGamepads()` call to be inside the throttled function. unless I'm missing something, that would still handle the Chrome case, since it's called on every `tick`.

Reply:
I thought the point of the throttle was to NOT call the throttled method every tick, but rather to enforce a minimum interval by ignoring calls that happen too soon since the last one.

Reply:
that’s what I’m suggesting. the code as is here checks for gamepads on every tick. can’t tell if I’m misunderstanding or not heh.

Reply:
This has to be tested across all browsers before being considered for merge. The reason navigator.getGamepads is called on every tick is because is needed in order to get updated poses for the controllers in some browser. The gamepad handle pose won’t update automatically and you see judder in the controller motion. Calling navigator.getGamepads might not be enough and retriving a new gamepad object might be necessary. That’s has to be determined after testing

Reply:
@dmarcos: awesome, thanks for the explanation. the pose information in each `Gamepad` object is the reason. 

in Chrome, `navigator.getGamepads()` is always an array of length 4. and just get an index from that array, and the `Gamepad` object is live. can’t use reference variables, obviously, as you know.

I think a possible improvement here would be to check for conencted/disconnected gamepads every 500 milliseconds, and have `tick` have separate checks for each individual `Gamepad` object.

that’s admittedly a lot of work for perhaps little benefit. any flare charts available to look at for some examples?